src/test/java/hudson/plugins/promoted_builds/conditions/ManualConditionTest.java

```diff
@@ -322,13 +351,26 @@ public void testManualPromotionPermissionsViaWebClient() throws Exception {
             assertThat(waitForBuildByNumber(pp, 3).getResult(), equalTo(Result.SUCCESS));
         }
 
+        {
+            // Re-execute promotion as specified user in varaible without Promotion/Promote
+            cond.setUsers("${approver}");
+            wc.login("non-promoter", "non-promoter");
+            try {
+                wc.getPage(b, String.format("promotion/%s/build?json={}", pp.getName()));
+                fail();
+            } catch (FailingHttpStatusCodeException e) {
+                assertThat(e.getStatusCode(), equalTo(404)); // Redirect after the build is broken.
+            }
+            assertThat(waitForBuildByNumber(pp, 4).getResult(), equalTo(Result.SUCCESS));
+        }
+
```

Comment:
@dwnusbaum 
Do you know why this webclient test fail?

Reply:
At a glance, no, but you could a debugger and step through the test/plugin code to figure out why it is failing before removing the test :)